src/pages/authentication/forms/ExamSetter.tsx

```
@@ -110,6 +157,24 @@
+      <div>
+        <Modal
+        title="Existing Organizations"
+        visible={showModal}
+        onOk={handleAddOrganization}
+        onCancel={() => setShowModal(false)}
+        okText="Add New Organization"
+      >
+        <p>You are already registered with the following organizations:</p>
+        <ul>
+          {setterOrganizations.map((org) => (
+            <li>{org.firstName}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)